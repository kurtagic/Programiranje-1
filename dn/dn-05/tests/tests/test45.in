753
3 410
3 18
3 305
3 48
3 211
3 -183
3 136
3 -290
3 -317
3 -296
3 500
3 -545
3 423
3 -625
3 -544
3 623
3 -707
3 570
3 -651
3 364
3 -511
3 -698
3 -260
3 233
3 -744
3 463
3 -449
3 -427
3 -641
3 44
3 -152
3 459
3 54
3 -634
3 469
3 -38
3 -326
3 430
3 319
3 -448
3 9
3 -527
3 13
3 -107
3 -384
3 -662
3 -418
3 373
3 -286
3 -578
3 -614
3 -28
3 -693
3 -287
3 -169
3 -173
3 -495
3 -355
3 -278
3 -185
3 42
3 -123
3 370
3 101
3 304
3 -12
3 534
3 -711
3 -713
3 229
3 -554
3 98
3 610
3 -207
3 72
3 -472
3 289
3 -23
3 508
3 659
3 242
3 -78
3 -32
3 430
3 -144
3 -585
3 208
3 -129
3 -312
3 531
3 -401
3 366
3 -395
3 449
3 37
3 417
3 -599
3 75
3 108
3 413
3 427
3 -366
3 -262
3 -338
3 -275
3 -245
3 484
3 301
3 602
3 43
3 469
3 111
3 372
3 223
3 -635
3 53
3 27
3 469
3 -565
3 145
3 -677
3 262
3 -149
3 -292
3 258
3 165
3 -472
3 -529
3 -136
3 40
3 -278
3 408
3 -153
3 -137
3 -227
3 -717
3 423
3 -3
3 509
3 450
3 -28
3 -205
3 -110
3 -363
3 301
3 642
3 234
3 -489
3 -68
3 -669
3 487
3 17
3 188
3 163
3 -733
3 29
3 473
3 -671
3 402
3 462
3 -140
3 -259
3 -592
3 -677
3 549
3 251
3 482
3 -559
3 484
3 151
3 -70
3 -735
3 -588
3 532
3 -399
3 657
3 -525
3 645
3 -210
3 440
3 -273
3 -4
3 32
3 283
3 -153
3 -481
3 -225
3 -731
3 107
3 12
3 -367
3 88
3 -529
3 641
3 -590
3 -328
3 538
3 338
3 322
3 -400
3 -377
3 -187
3 -675
3 444
3 -456
3 662
3 594
3 202
3 -92
3 50
3 -343
3 -440
3 -532
3 460
3 136
3 109
3 456
3 -729
3 30
3 -714
3 220
3 -267
3 -401
3 290
3 330
3 -502
3 -179
3 374
3 67
3 -527
3 510
3 470
3 -660
3 -214
3 481
3 -23
3 -520
3 -651
3 36
3 -40
3 -305
3 -546
3 -295
3 -600
3 -635
3 72
3 -333
3 189
3 -245
3 -709
3 -434
3 -294
3 356
3 247
3 476
3 -259
3 -421
3 444
3 -85
3 -43
3 -422
3 -137
3 -664
3 335
3 367
3 345
3 -718
3 249
3 -722
3 249
3 654
3 -235
3 536
3 -120
3 -530
3 603
3 308
3 -661
3 366
3 -533
3 -71
3 -553
3 -628
3 304
3 -661
3 153
3 -157
3 -528
3 -423
3 569
3 -663
3 -721
3 -684
3 -206
3 141
3 593
3 363
3 226
3 20
3 343
3 -463
3 -567
3 -129
3 582
3 -29
3 132
3 459
3 256
3 -30
3 -489
3 617
3 -65
3 -574
3 202
3 -407
3 -26
3 -355
3 98
3 121
3 119
3 556
3 -549
3 -245
3 480
3 521
3 404
3 -353
3 -461
3 -601
3 246
3 194
3 628
3 -455
3 223
3 249
3 -618
3 -295
3 206
3 212
3 -256
3 195
3 -225
3 -592
3 -593
3 -99
3 -446
3 -298
3 197
3 342
3 -202
3 -475
3 -38
3 117
3 11
3 -517
3 -587
3 -539
3 160
3 53
3 544
3 -558
3 405
3 513
3 343
3 -208
3 -18
3 305
3 289
3 -150
3 -339
3 108
3 -163
3 -39
3 45
3 329
3 163
3 -729
3 279
3 -86
3 -420
3 367
3 121
3 -421
3 -135
3 -724
3 387
3 12
3 135
3 385
3 317
3 215
3 642
3 -134
3 91
3 327
3 -484
3 73
3 -672
3 -475
3 378
3 471
3 560
3 -409
3 -180
3 49
3 351
3 -516
3 -199
3 129
3 -551
3 62
3 509
3 32
3 -30
3 251
3 -245
3 578
3 -345
3 34
3 577
3 -438
3 -123
3 -187
3 -383
3 97
3 80
3 231
3 -616
3 445
3 -652
3 250
3 -223
3 247
3 -179
3 213
3 390
3 -725
3 -56
3 -297
3 -687
3 138
3 -52
3 146
3 216
3 382
3 -72
3 145
3 -63
3 77
3 -151
3 -665
3 631
3 113
3 -364
3 235
3 479
3 281
3 461
3 -205
3 199
3 -364
3 605
3 -668
3 468
3 -701
3 -282
3 359
3 244
3 -526
3 -638
3 -86
3 5
3 77
3 185
3 -260
3 -121
3 554
3 595
3 555
3 -503
3 -646
3 -491
3 -591
3 144
3 463
3 458
3 614
3 656
3 25
3 -740
3 -24
3 -384
3 361
3 273
3 -212
3 -525
3 598
3 -547
3 -269
3 403
3 -730
3 -60
3 516
3 -165
3 612
3 -92
3 -54
3 460
3 -140
3 498
3 489
3 246
3 -671
3 447
3 -251
3 -495
3 247
3 -478
3 234
3 51
3 -351
3 -388
3 -109
3 588
3 -218
3 210
3 274
3 -498
3 -653
3 87
3 135
3 -508
3 -656
3 -433
3 -457
3 -236
3 -407
3 359
3 -18
3 156
3 -431
3 60
3 274
3 624
3 -469
3 -695
3 403
3 -537
3 507
3 -94
3 -75
3 -362
3 -596
3 264
3 -89
3 -259
3 192
3 -494
3 -454
3 -113
3 -547
3 589
3 527
3 213
3 79
3 -524
3 48
3 -29
3 -610
3 160
3 -658
3 -92
3 -582
3 627
3 -277
3 -721
3 -312
3 209
3 469
3 443
3 82
3 -168
3 -365
3 -374
3 474
3 -532
3 -717
3 121
3 -234
3 247
3 -442
3 -339
3 -570
3 -111
3 -595
3 195
3 -377
3 -389
3 434
3 209
3 632
3 1
3 438
3 42
3 69
3 -37
3 -628
3 -18
3 242
3 204
3 573
3 -388
3 -242
3 -514
3 -59
3 -129
3 -724
3 187
3 -195
3 658
3 18
3 -60
3 -452
3 -359
3 502
3 -17
3 -16
3 385
3 -351
3 11
3 -611
3 -589
3 415
3 -670
3 -410
3 -133
3 -263
3 -374
3 366
3 285
3 142
3 41
3 373
3 -674
3 101
3 327
3 316
3 275
3 578
3 313
3 -48
3 -429
3 292
3 119
3 -9
3 -23
3 -650
3 44
3 -183
3 -260
3 484
3 266
3 -409
3 -154
3 620
3 375
3 -159
3 217
3 230
3 -260
3 -546
3 122
3 -674
3 -134
3 371
3 -390
3 -590
3 626
3 574
3 -8
3 -92
3 -678
3 49
3 -593
3 10
3 -61
3 55
3 112
3 -499
3 153
3 -338
3 -266
3 -716
3 132
3 486
3 -147
3 316
3 -397
3 436
3 -281
3 -655
3 270
3 -664
3 229
3 -696
3 646
3 -46
3 -14
3 -162
3 -334
3 -269
3 -227
3 619
3 17
3 -454
3 553
3 -287
3 -438
3 314
3 -627
3 -462
3 556
3 -417
3 -100
3 -312
3 639
3 181
3 79
3 359
3 64
3 50
3 498
3 -413
3 449
3 172
3 657
3 347
3 -740
3 615
3 446
3 255
3 65
3 0
3 -281
3 -80
3 523
3 81
3 211
3 -477
3 625
3 -123
3 407
3 -547
