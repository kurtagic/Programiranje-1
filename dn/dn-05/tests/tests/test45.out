-744: 1
-743: 2
-742: 1
-740: 3
-739: 4
-738: 3
-735: 2
-734: 3
-733: 2
-732: 2
-731: 2
-730: 7
-729: 8
-728: 7
-727: 5
-725: 4
-724: 6
-723: 7
-722: 6
-721: 5
-720: 6
-719: 5
-718: 3
-717: 5
-716: 7
-715: 8
-714: 7
-713: 5
-712: 4
-711: 2
-710: 2
-709: 2
-708: 3
-707: 2
-706: 2
-705: 1
-701: 1
-700: 2
-699: 1
-698: 1
-697: 2
-696: 4
-695: 5
-694: 4
-693: 2
-692: 2
-691: 1
-687: 1
-686: 2
-685: 1
-684: 1
-683: 2
-682: 1
-678: 5
-677: 6
-676: 5
-675: 4
-674: 8
-673: 9
-672: 8
-671: 6
-670: 8
-669: 9
-668: 8
-667: 4
-666: 3
-665: 8
-664: 10
-663: 11
-662: 10
-661: 7
-660: 7
-659: 6
-658: 3
-657: 4
-656: 3
-655: 8
-654: 9
-653: 8
-652: 8
-651: 7
-650: 6
-649: 7
-648: 6
-646: 1
-645: 2
-644: 1
-641: 1
-640: 2
-639: 1
-638: 1
-637: 2
-636: 1
-635: 5
-634: 6
-633: 5
-632: 1
-628: 3
-627: 5
-626: 6
-625: 5
-624: 2
-623: 1
-618: 1
-617: 2
-616: 2
-615: 3
-614: 2
-613: 2
-612: 1
-611: 3
-610: 4
-609: 3
-608: 1
-601: 5
-600: 6
-599: 5
-598: 3
-597: 1
-596: 1
-595: 6
-594: 7
-593: 7
-592: 8
-591: 7
-590: 9
-589: 10
-588: 9
-587: 7
-586: 4
-585: 3
-584: 2
-583: 1
-582: 1
-581: 2
-580: 1
-578: 1
-577: 2
-576: 1
-574: 1
-573: 2
-572: 1
-570: 1
-569: 2
-568: 1
-567: 2
-566: 3
-565: 2
-564: 2
-563: 1
-559: 1
-558: 3
-557: 4
-556: 3
-554: 1
-553: 3
-552: 4
-551: 4
-550: 5
-549: 4
-548: 2
-547: 12
-546: 13
-545: 12
-544: 10
-543: 4
-542: 3
-539: 1
-538: 2
-537: 2
-536: 3
-535: 2
-533: 3
-532: 7
-531: 8
-530: 7
-529: 7
-528: 7
-527: 8
-526: 8
-525: 10
-524: 12
-523: 13
-522: 12
-520: 1
-519: 2
-518: 1
-517: 1
-516: 3
-515: 4
-514: 4
-513: 5
-512: 4
-511: 1
-510: 2
-509: 1
-508: 1
-507: 2
-506: 1
-503: 3
-502: 4
-501: 3
-500: 1
-499: 3
-498: 4
-497: 3
-496: 1
-495: 3
-494: 5
-493: 6
-492: 5
-491: 4
-490: 5
-489: 4
-488: 4
-487: 3
-484: 1
-483: 2
-482: 1
-481: 1
-480: 2
-479: 1
-478: 1
-477: 4
-476: 5
-475: 4
-474: 4
-473: 3
-472: 3
-471: 4
-470: 3
-469: 1
-468: 2
-467: 1
-463: 1
-462: 4
-461: 5
-460: 4
-459: 2
-457: 4
-456: 5
-455: 4
-454: 7
-453: 8
-452: 7
-451: 7
-450: 6
-449: 1
-448: 3
-447: 4
-446: 4
-445: 5
-444: 4
-442: 2
-441: 3
-440: 2
-439: 2
-438: 4
-437: 5
-436: 4
-434: 1
-433: 3
-432: 4
-431: 4
-430: 5
-429: 5
-428: 6
-427: 5
-426: 2
-425: 1
-423: 5
-422: 6
-421: 6
-420: 7
-419: 6
-418: 4
-417: 3
-416: 4
-415: 3
-413: 1
-412: 2
-411: 1
-410: 4
-409: 6
-408: 7
-407: 6
-406: 4
-405: 3
-401: 6
-400: 7
-399: 6
-398: 4
-397: 3
-396: 4
-395: 3
-394: 2
-393: 1
-390: 6
-389: 7
-388: 6
-387: 6
-386: 5
-384: 5
-383: 6
-382: 5
-381: 3
-377: 3
-376: 4
-375: 3
-374: 3
-373: 4
-372: 3
-367: 3
-366: 4
-365: 7
-364: 8
-363: 7
-362: 6
-361: 7
-360: 6
-359: 1
-358: 2
-357: 1
-355: 3
-354: 4
-353: 4
-352: 5
-351: 7
-350: 8
-349: 7
-345: 2
-344: 3
-343: 2
-342: 2
-341: 1
-339: 5
-338: 7
-337: 8
-336: 7
-334: 3
-333: 4
-332: 3
-331: 1
-328: 2
-327: 3
-326: 2
-325: 2
-324: 1
-317: 1
-316: 2
-315: 1
-312: 5
-311: 6
-310: 5
-305: 1
-304: 2
-303: 1
-298: 2
-297: 8
-296: 9
-295: 8
-294: 8
-293: 7
-292: 5
-291: 3
-290: 2
-289: 2
-288: 1
-287: 5
-286: 6
-285: 5
-284: 1
-282: 1
-281: 5
-280: 6
-279: 5
-278: 3
-277: 5
-276: 6
-275: 5
-274: 2
-273: 2
-272: 3
-271: 2
-269: 4
-268: 5
-267: 4
-266: 3
-265: 4
-264: 3
-263: 4
-262: 5
-261: 4
-260: 13
-259: 14
-258: 13
-257: 9
-256: 1
-255: 2
-254: 1
-251: 1
-250: 2
-249: 1
-245: 7
-244: 8
-243: 7
-242: 1
-241: 2
-240: 1
-236: 3
-235: 4
-234: 4
-233: 5
-232: 4
-227: 5
-226: 6
-225: 5
-224: 5
-223: 5
-222: 6
-221: 5
-218: 1
-217: 2
-216: 1
-214: 1
-213: 2
-212: 2
-211: 3
-210: 2
-209: 2
-208: 5
-207: 6
-206: 5
-205: 6
-204: 7
-203: 6
-202: 1
-201: 2
-200: 1
-199: 1
-198: 2
-197: 1
-195: 1
-194: 2
-193: 1
-187: 5
-186: 6
-185: 5
-184: 3
-183: 3
-182: 4
-181: 3
-180: 3
-179: 5
-178: 6
-177: 5
-173: 1
-172: 2
-171: 1
-169: 1
-168: 3
-167: 4
-166: 3
-165: 2
-164: 3
-163: 2
-162: 3
-161: 4
-160: 3
-159: 2
-158: 3
-157: 2
-156: 2
-155: 1
-154: 7
-153: 8
-152: 7
-151: 6
-150: 7
-149: 6
-148: 3
-147: 2
-146: 3
-145: 2
-144: 1
-143: 2
-142: 1
-140: 3
-139: 4
-138: 3
-137: 5
-136: 6
-135: 6
-134: 12
-133: 13
-132: 12
-131: 10
-129: 5
-128: 6
-127: 5
-123: 5
-122: 6
-121: 5
-120: 5
-119: 4
-118: 1
-113: 1
-112: 2
-111: 4
-110: 5
-109: 4
-108: 4
-107: 3
-106: 2
-105: 1
-100: 3
-99: 4
-98: 3
-97: 1
-94: 4
-93: 5
-92: 7
-91: 8
-90: 7
-89: 1
-88: 2
-87: 1
-86: 5
-85: 6
-84: 5
-83: 1
-80: 2
-79: 3
-78: 2
-77: 2
-76: 1
-75: 1
-74: 2
-73: 1
-72: 6
-71: 7
-70: 6
-69: 4
-68: 2
-67: 2
-66: 1
-65: 1
-64: 2
-63: 2
-62: 3
-61: 7
-60: 8
-59: 7
-58: 5
-57: 3
-56: 1
-55: 2
-54: 2
-53: 3
-52: 2
-51: 2
-50: 1
-48: 1
-47: 2
-46: 2
-45: 3
-44: 2
-43: 1
-42: 2
-41: 1
-40: 2
-39: 5
-38: 6
-37: 6
-36: 7
-35: 6
-32: 1
-31: 2
-30: 9
-29: 11
-28: 12
-27: 11
-26: 4
-25: 5
-24: 5
-23: 7
-22: 8
-21: 7
-18: 5
-17: 7
-16: 9
-15: 10
-14: 10
-13: 11
-12: 10
-11: 2
-10: 1
-9: 1
-8: 3
-7: 4
-6: 3
-4: 3
-3: 4
-2: 3
-1: 1
0: 3
1: 4
2: 3
3: 1
5: 1
6: 2
7: 1
9: 1
10: 11
11: 12
12: 11
13: 9
14: 7
15: 1
17: 7
18: 8
19: 7
20: 5
21: 3
22: 2
25: 2
26: 3
27: 2
28: 2
29: 2
30: 4
31: 5
32: 5
33: 6
34: 6
35: 7
36: 6
37: 4
38: 3
39: 1
40: 3
41: 8
42: 9
43: 8
44: 7
45: 8
46: 7
47: 5
48: 6
49: 8
50: 10
51: 11
52: 10
53: 6
54: 6
55: 6
56: 7
57: 6
60: 2
61: 3
62: 2
63: 2
64: 2
65: 4
66: 5
67: 4
68: 2
69: 2
70: 3
71: 2
72: 3
73: 5
74: 6
75: 5
76: 2
77: 4
78: 5
79: 7
80: 8
81: 8
82: 9
83: 8
84: 2
87: 3
88: 4
89: 3
90: 1
91: 1
92: 2
93: 1
97: 5
98: 6
99: 5
100: 3
101: 3
102: 4
103: 3
107: 3
108: 6
109: 7
110: 6
111: 4
112: 4
113: 5
114: 4
115: 2
117: 3
118: 4
119: 6
120: 7
121: 6
122: 7
123: 8
124: 7
129: 1
130: 2
131: 1
132: 3
133: 4
134: 3
135: 7
136: 8
137: 7
138: 4
139: 5
140: 4
141: 1
142: 8
143: 9
144: 8
145: 8
146: 7
147: 5
148: 3
151: 1
152: 2
153: 4
154: 5
155: 4
156: 1
157: 2
158: 1
160: 3
161: 4
162: 3
163: 4
164: 5
165: 4
166: 2
167: 1
172: 1
173: 2
174: 1
181: 1
182: 2
183: 1
185: 1
186: 2
187: 4
188: 5
189: 4
190: 4
191: 3
192: 2
193: 3
194: 2
195: 5
196: 6
197: 5
198: 5
199: 5
200: 6
201: 5
202: 3
203: 4
204: 4
205: 5
206: 4
207: 3
208: 2
209: 8
210: 9
211: 9
212: 10
213: 9
214: 8
215: 7
216: 3
217: 5
218: 6
219: 5
220: 1
221: 2
222: 1
223: 3
224: 4
225: 3
226: 1
227: 2
228: 1
229: 6
230: 7
231: 6
232: 4
233: 2
234: 7
235: 8
236: 7
237: 5
242: 5
243: 6
244: 5
245: 5
246: 9
247: 13
248: 14
249: 13
250: 9
251: 10
252: 9
253: 7
255: 4
256: 5
257: 4
258: 2
259: 2
260: 1
262: 1
263: 2
264: 2
265: 3
266: 3
267: 4
268: 3
270: 1
271: 2
272: 1
273: 1
274: 5
275: 7
276: 8
277: 7
279: 1
280: 2
281: 2
282: 3
283: 2
284: 2
285: 2
286: 3
287: 2
289: 5
290: 6
291: 5
292: 4
293: 5
294: 4
301: 3
302: 4
303: 3
304: 5
305: 7
306: 8
307: 7
308: 1
309: 2
310: 1
313: 1
314: 7
315: 8
316: 7
317: 7
318: 6
319: 2
320: 2
321: 1
322: 1
323: 2
324: 1
327: 6
328: 7
329: 6
330: 4
331: 3
332: 1
335: 1
336: 2
337: 1
338: 1
339: 2
340: 1
342: 4
343: 6
344: 7
345: 6
346: 2
347: 2
348: 3
349: 2
351: 1
352: 2
353: 1
356: 1
357: 2
358: 1
359: 7
360: 8
361: 7
362: 5
363: 4
364: 4
365: 3
366: 10
367: 11
368: 10
369: 8
370: 1
371: 7
372: 8
373: 7
374: 7
375: 7
376: 8
377: 7
378: 1
379: 2
380: 1
382: 1
383: 2
384: 1
385: 4
386: 5
387: 4
388: 2
389: 1
390: 1
391: 2
392: 1
402: 1
403: 7
404: 8
405: 7
406: 5
407: 5
408: 6
409: 5
410: 2
411: 2
412: 1
413: 1
414: 2
415: 2
416: 3
417: 2
418: 2
419: 1
423: 3
424: 4
425: 3
427: 1
428: 2
429: 1
430: 3
431: 4
432: 3
434: 1
435: 2
436: 3
437: 4
438: 3
439: 3
440: 2
441: 2
442: 1
443: 6
444: 7
445: 6
446: 8
447: 9
448: 8
449: 7
450: 8
451: 7
452: 3
456: 1
457: 2
458: 8
459: 9
460: 9
461: 10
462: 9
463: 8
464: 9
465: 8
468: 8
469: 10
470: 11
471: 10
472: 10
473: 9
474: 3
475: 4
476: 3
477: 2
478: 1
479: 8
480: 9
481: 8
482: 6
483: 4
484: 5
485: 6
486: 6
487: 7
488: 6
489: 2
490: 3
491: 2
498: 4
499: 5
500: 4
501: 2
502: 2
503: 3
504: 2
507: 8
508: 9
509: 8
510: 8
511: 7
512: 5
513: 1
514: 2
515: 1
516: 1
517: 2
518: 1
521: 1
522: 2
523: 2
524: 3
525: 2
527: 1
528: 2
529: 1
531: 1
532: 3
533: 4
534: 3
535: 2
536: 2
537: 3
538: 2
539: 2
540: 1
544: 1
545: 2
546: 1
549: 1
550: 2
551: 1
553: 5
554: 6
555: 5
556: 6
557: 7
558: 6
560: 1
561: 2
562: 1
569: 3
570: 4
571: 3
572: 1
573: 1
574: 3
575: 4
576: 3
577: 3
578: 5
579: 6
580: 5
582: 1
583: 2
584: 1
588: 1
589: 3
590: 4
591: 3
593: 3
594: 4
595: 4
596: 5
597: 4
598: 1
599: 2
600: 1
602: 1
603: 3
604: 4
605: 4
606: 5
607: 4
610: 1
611: 2
612: 2
613: 3
614: 2
615: 3
616: 4
617: 3
618: 2
619: 3
620: 4
621: 3
622: 1
623: 1
624: 3
625: 7
626: 8
627: 7
628: 5
629: 3
630: 1
631: 1
632: 3
633: 4
634: 3
639: 4
640: 5
641: 4
642: 5
643: 6
644: 5
645: 1
646: 3
647: 4
648: 3
654: 1
655: 2
656: 4
657: 7
658: 8
659: 7
660: 5
661: 1
662: 1
663: 2
664: 1
